### DISCUSSION TITLE
Disable Sass

### DISCUSSION THREAD

**Discussion by ankitshekhawat** [hayes0724/shopify-packer#105] [Q&A] [answered]
How do I disable Sass and only use css + postcss

**Comment by hayes0724** [comment, marked as answer]
You would need to build a custom webpack override, it's not well documented currently:

https://hayes0724.github.io/shopify-packer/docs/config/webpack

You will need to look at the source prod and dev webpack config in packer to see what to override.

I also have plans for each "core" webpack part (scss, css, entrypoints, etc..) to have a toggle to disable each easily in config.js

**Reply by ankitshekhawat** [reply]
Thanks, @hayes0724, for now, I just changed the entry point to a CSS file instead of a `theme.scss`  in `theme.js` . That seems to do the job. Sass is still part of the workflow but doesn't get activated.
